draft-duke-quic-protected-initial.md

```diff
@@ -490,9 +490,19 @@ to a version of QUIC that does not protect Initials. Endpoints MUST implement
 {{!I-D.ietf-quic-version-negotiation}} if they support multiple versions to
 mitigate this attack.
 
-Note that detection does not occur until after the client has sent a new,
-unprotected Initial. When composing this Initial, the client should therefore
-consider the points in {{downgrade}} below.
+Note that endpoints do not detect the attack until after the client has sent a
+new, unprotected Initial.
+
+Upon receipt of a Version Negotation packet, a client SHOULD wait for a probe
+timeout (PTO) before sending an Initial using another version. If it receives a
+valid Server Initial, it SHOULD ignore the Version Negotation. This eliminates
```

Comment:
This SHOULD is a MUST in QUICv1 so we don't need to say anything. Or if we want to point it out, we should reference that MUST.